CODE_STYLE.md

```diff
@@ -11,11 +11,34 @@ Some of the rules are enforced with static tests. You can read the [TESTING][1]
 document for more information and details about how to run the static suite of
 tests. Some other rules are only socially enforced during code reviews.
 
-## black
+## Code Formatting
 
 This code base adheres to black[2].
 You can `snap install black --beta --devmode` to install the linter and formatter onto your host.
 
+## Conditionals
+
+* Always check for expected value e.g.; `if foo is True` instead of `if foo`
+  or `if foo is not None`.
+  
+## Methods
+
+* Return only once from a method unless it is through a guard.
```

Comment:
I'm still not entirely sure about this one, but I have been thinking about it... it's worth giving it a go anyhow

Reply:
here are some words stringed together that might convince you https://docs.python-guide.org/writing/style/#returning-values :-)